model/log/empty_iterator.go

```diff
@@ -0,0 +1,19 @@
+package log
+
+var empty = &emptyIterator{}
+
+type emptyIterator struct{}
+
+// EmptyIterator returns a convenience log iterator with no data that returns
+// false immediately.
```

Comment:
Can you disambiguate what `return false` mean in this case? It already says that it holds "no data" so it seems like that would be sufficient explanation.

Reply:
Oh that was referring to the `Next` command, I will just remove it.